.github/workflows/claude.yml

```
@@ -0,0 +1,58 @@
+      issues: read
+      id-token: write
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 1
+
+      - name: Run Claude Code
+        id: claude
+        uses: anthropics/claude-code-action@beta
+        with:
```

actionlint (1.7.7)

"with" section should not be empty. please remove this section if it's unnecessary

(syntax-check)